packages/ui/utils/formatBreadcrumbsPath.js

```diff
@@ -0,0 +1,10 @@
+import { capitalizeFirstLetter } from '@/utils/capitalizeFirstLetter'
```

Comment:
What do you think about creating a `breadcrumbs/` folder move the components and this utility function there?